### PR TITLE
Add properties to schema

### PR DESCRIPTION
**Pull request by jasollien**

### CODE REVIEW

**Comment by jasollien** [comment]
@pasi-paasiala, @ykulbak , @GeorgDangl: Could you have a look at this, and give some feedback. I have tried making it based on your ideas.

**Comment by ykulbak** [comment]
@jasollien I think it's a good start. I think we should also allow extension schema information regarding the allowed values for some properties and possibly topic authorisation information regarding changing the values of properties.

I've spent a few minutes reviewing and I need more time to think about this.

**Comment by pasi-paasiala** [comment]
I think we also need to have enumeration properties and be able to specify physical quantity (like length) to numerical properties.

**Review by GeorgDangl — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -379,9 +379,20 @@ Project extensions are used to define possible values that can be used in topics
...
+                "id": "6e6bddaa-4c53-4fb8-b884-500e0d2dba6a",
+                "name": "price_in_dollar",
+                "type": "decimal",
+                "readonly": false,
+                "required": true
+            }
         ]
     }
 
+> **About custom fields:** Projects may be configured to allow `custom_fields` in a topic. This is a way of enabling clients and servers to embed custom data in a topic. Those custom fields can be sent when creating or updating a topic, and they will be returned by the server when retrieving topics.
```
Comment (jasollien) [inline README.md:394]:
```suggestion
> **About custom fields:** Projects may be configured to allow `custom_fields` in a topic. This is a way of enabling clients and servers to embed custom data in a topic. Those custom fields should be sent when creating or updating a topic, and they will be returned by the server when retrieving topics.
```

**Pull request merged**